### PR TITLE
create first draft of GSoC project post

### PR DESCRIPTION
**Pull request by kain88-de**
This is a first draft of the GSoC announcement post. I will add more information about the BIO from @jdetle and @fiona-naughton when I get it and pictures is you like to have them posted here as well.

The project description could use some work still though.

ping @MDAnalysis/coredevs

### CODE REVIEW

File: _posts/2016-04-26-gsoc.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: GSoC Projects
+---
+
+We are happy to anounce that MDAnalysis is hosting two GSoC students this year.
+Fiona Naughton and John Detlefs.
+
+# Dimension Reduction
```
Comment (orbeckst) [inline _posts/2016-04-26-gsoc.md]: Add name to heading:
```
# John Detlefs: Dimensionality Reduction

# Fiona Naughton: Analysis of umbrella sampling simulations

```

**Comment by orbeckst** [comment]
* Add links to the official GsoC announcements
  * [jdetle](https://summerofcode.withgoogle.com/organizations/4890191244296192/#5875821348651008)
  * [fiona](https://summerofcode.withgoogle.com/organizations/4890191244296192/#5634029508362240)
* Add GSoC and PSF logos and acknowledge them with links.

**Comment by kain88-de** [comment]
I make the changes you suggested. The pictures at the top aren't aligned very pretty though.

As pictures I have right now just used the github avatars as placeholders. They are placed next the to the heading but placed a little bit to high. Maybe someone with more experience in html can fix that, (maybe @dotsdl )

**Comment by jdetle** [comment]
Bio:

John Detlefs is a Mathematics and Chemistry double major at California Polytechnic State University, San Luis Obispo.  When not contributing to MDAnalysis, John can be found reading a good book or enjoying one of the many outdoor activities California has to offer. After graduating in June 2016, John plans on pursuing a career in scientific computing.

avatar

**Comment by jbarnoud** [comment]
@kain88-de Would it help to have the two logos in a `<p>` tag? There should then be a line break after them, which could fix the alignment issues.

**Comment by kain88-de** [comment]
I'm not sure but I can try.

**Comment by orbeckst** [comment]
@jdetle do you have any links to be included, e.g. home page/blog/twitter?

**Comment by jdetle** [comment]
jdetle.com, that will be where I will maintain my GSoC blog and it links to
my Twitter.
On Sun, Apr 24, 2016 at 11:10 PM Oliver Beckstein <notifications@github.com>
wrote:

> @jdetle <https://github.com/jdetle> do you have any links to be included,
> e.g. home page/blog/twitter?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/MDAnalysis/MDAnalysis.github.io/pull/24#issuecomment-214153346>
>
-- 
Have a wonderful day,
John J. Detlefs

**Comment by kain88-de** [comment]
@jbarnoud the paragraph tag doesn't help

**Comment by fiona-naughton** [comment]
Here's my bio (sorry for the delay -_- ). I'm happy with leaving my github avatar for the picture.

Fiona Naughton is doing a PhD in Biochemistry at the University of Oxford with the Structural Bioinformatics and Computational Biochemistry Unit, studying protein/membrane interactions through molecular dynamics simulations. She plans to continue to a career in the same field and outside academia enjoys reading, baking and handicraft.

**Comment by richardjgowers** [comment]
Ok added the bios, time to merge?

File: _posts/2016-04-26-gsoc.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+In the next weeks they will both further refine their projects and setup
+personal blogs to update anyone interested about their progress during the
+summer. We plan to have all discussions public on the
+[devel mailinglist][devel].
+
+--- @kain88-de
+
+[pca]: https://en.wikipedia.org/wiki/Principal_component_analysis
+[dm]: https://en.wikipedia.org/wiki/Diffusion_map
+[john-blog]: https://jdetle.com
```
Comment (kain88-de) [inline _posts/2016-04-26-gsoc.md]: the blog is not https

Comment (richardjgowers) [inline _posts/2016-04-26-gsoc.md]: K fixed

**Comment by kain88-de** [comment]
I've updated the picture to look a little bit nicer formatted. 

I wanted to check the project description in the evening again. Otherwise I'm good with it.

File: _posts/2016-04-26-gsoc.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+different observables.
+
+Fiona Naughton is doing a PhD in Biochemistry at the University of Oxford with
+the Structural Bioinformatics and Computational Biochemistry Unit, studying
+protein/membrane interactions through molecular dynamics simulations. She has
+started her own [blog][fiona-blog] and can be found in github under
+@fiona-naughton. She plans to continue to a career in the same field and outside
+academia enjoys reading, baking and handicraft.
+
+# John Detlefs: Dimension Reduction
```
Comment (orbeckst) [inline _posts/2016-04-26-gsoc.md]: Dimensionality Reduction

File: _posts/2016-04-26-gsoc.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+<img
+src="https://avatars1.githubusercontent.com/u/10944582?v=3&s=460"
+title="John Detlefs" alt="John Detlefs"
+style="float: left; width: 110px; height: 110px; border-radius: 20px; border: 15px solid white" />
+
+John is working on adding Dimension reduction algorithms to our analysis module.
```
Comment (orbeckst) [inline _posts/2016-04-26-gsoc.md]: Dimensionality Reduction

File: _posts/2016-04-26-gsoc.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+He has chosen two algorithms that work purely on the data without additional
+input, [principle component analysis][pca] and [diffusion maps][dm].
```
Comment (orbeckst) [inline _posts/2016-04-26-gsoc.md]: principal

File: _posts/2016-04-26-gsoc.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+simulations in which a *reaction coordinate* , such as the distance between two
+molecules, is restrained to different values. A method to analyse the
+combination of the simulations is [weighted histogram analysis method][wham]
+(WHAM). Fiona will work on adding WHAM to the analysis module to calculate
+different observables.
+
+Fiona Naughton is doing a PhD in Biochemistry at the University of Oxford with
+the Structural Bioinformatics and Computational Biochemistry Unit, studying
+protein/membrane interactions through molecular dynamics simulations. She has
+started her own [blog][fiona-blog] and can be found in github under
```
Comment (orbeckst) [inline _posts/2016-04-26-gsoc.md]: URL for Fiona's blog needed – @fiona-naughton can you supply a working UR, please?

**Comment by orbeckst** [comment]
See comments, I think "*dimensionality* reduction" is more common and it's definitely called "*principal* component analysis."

**Comment by kain88-de** [comment]
Good to go from my side. Please publish it when you feel like it. I will be going out today so I won't answer before tomorrrow.

**Pull request merged**